core/pom.xml

```diff
@@ -772,6 +772,7 @@ THE SOFTWARE.
           <forkCount>0.5C</forkCount>
           <reuseForks>true</reuseForks>
           <argLine>-XX:MaxPermSize=128m -noverify</argLine> <!-- some versions of JDK7/8 causes VerifyError during mock tests: http://code.google.com/p/powermock/issues/detail?id=504 -->
+          <trimStackTrace>false</trimStackTrace> <!-- SUREFIRE-1226 workaround -->
```

Comment:
Otherwise the pre-fix test failure omits the interesting bits in its stack trace.